qemu/tests/hpt_max_page_size.py

```diff
@@ -0,0 +1,34 @@
+from virttest import error_context
+from virttest.staging.utils_memory import read_from_meminfo
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test to maximum page size for hash page table guests.
+    Steps:
+    1) There are two options, with/without mem backing file,
+       if without mem backing, ignore the step 3.
+    2) System setup hugepages on host.
+    3) Mount this hugepage to /mnt/kvm_hugepage.
+    4) Boot up guest options with different cap-hpt-max-page-size,
+       and there are two modes with kvm or tcg.
+    5) Check the hugepage size is 0kB inside guest.
+    6) Check the output parameter has only two option,
+       when the cap-hpt-max-page-size is 64k.
+
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+
+    if any(read_from_meminfo(_, session)
+           for _ in params.objects('hp_check_keys')):
+        test.fail("Not all the HugePages size are 0kB.")
```

Comment:
Print the memory information before `test.fail`, it is useful for debugging.

Reply:
@PaulYuuu  I appreciate your help, @MiriamDeng Do you need to print this information or not ?

Reply:
zhenyzha,it is a good idea if there's output while debugging in my opinions.